src/dbally/prompts/input_format.py

```diff
@@ -0,0 +1,91 @@
+import copy
+from abc import abstractmethod
+from typing import Dict, List, Tuple
+
+from dbally.prompts.few_shot import FewShotExample
+from dbally.prompts.prompt_template import PromptTemplate
+from dbally.views.exposed_functions import ExposedFunction
+
+
+def _promptify_filters(
+    filters: List[ExposedFunction],
+) -> str:
+    """
+    Formats filters for prompt
+
+    Args:
+        filters: list of filters exposed by the view
+
+    Returns:
+        filters_for_prompt: filters formatted for prompt
+    """
+    filters_for_prompt = "\n".join([str(filter) for filter in filters])
+    return filters_for_prompt
+
+
+class AbstractInputFormatter:
```

Comment:
I think it should be an abstract class, also don't use "Abstract" in the class name, just call it `InputFormatter`.

Reply:
thank you, I agree